maat-court-data-api/src/main/java/gov/uk/courtdata/contribution/service/ContributionsService.java

```diff
@@ -57,4 +57,9 @@ public ContributionsDTO create(CreateContributions createContributions) {
         newContributionsEntity.setLatest(true);
         return contributionsMapper.mapEntityToDTO(contributionsRepository.saveAndFlush(newContributionsEntity));
     }
+
+    @Transactional
+    public Integer getContributionCount(Integer repId) {
```

Comment:
The return type here should probably be `int` as the count will always be a non-null integer >0